projects/cdk/utils/dom/retarget-boundary-crossing.ts

```diff
@@ -0,0 +1,15 @@
+export function tuiRetargetedBoundaryCrossing(event: Event): boolean {
```

Comment:
Looks like we should just do `event: any`

Reply:
fixed